src/Reporter/Reporter.php

```diff
@@ -35,21 +35,21 @@ public function __construct()
 
     public function progress($current, $total)
     {
-        echo sprintf("Reviewing file %d of %d.\r", $current, $total);
+        echo sprintf("Reviewing %d of %d." . PHP_EOL, $current, $total);
```

Comment:
I'm hesitant to remove the `\r` return here, it's clearing the line in the console and potentially stopping a flood of messages.

While I don't feel this class should really be echoing anything it'd be good to remain consistent until it gets a refactor.